projects/client/Unit/src/unit/APIApproval.Approve.approved.txt

```diff
@@ -15,6 +15,7 @@ namespace RabbitMQ.Client
         public int Port { get; set; }
         public RabbitMQ.Client.IProtocol Protocol { get; }
         public RabbitMQ.Client.SslOption Ssl { get; set; }
+        public RabbitMQ.Client.SslOption Tls { get; set; }
```

Comment:
What's going on here? I don't see anything in the PR that would cause this to show up.

Reply:
I have added this alias in `master` as part of the initial investigation of #764 where I added some more URI parsing tests. `APIApproval` expectations were not updated there, I will do so now. So the code change does not show up here because it's already in master but the `APIApproval` change is not.

Reply:
Can you remove it instead? Not sure why we need two different values here.

Reply:
Done. My dream of eradicating "SSL" everywhere we can in favor of "TLS" is still alive, though 😆